### DISCUSSION TITLE
Is there a way to do it, integrate it in my app and install it when I install my app

### DISCUSSION THREAD

**Discussion by jackwxs** [ExistentialAudio/BlackHole#762] [Q&A]
Is there a way to do it, integrate it in my app and install it when I install my app

**Comment by jackwxs** [comment]
Because now I need to install it first and then install my program. This is equivalent to installing 2 programs.

**Comment by devinroth** [comment]
Email me at ***@***.*** regarding getting a valid license to use and bundle blackhole with a non-GPL program and I will give you all the information you require.

Devin
